banyan/src/zstd_array.rs

```diff
@@ -134,7 +179,8 @@ impl ZstdArray {
                 }
             }
             Ok(result)
-        })?
+        })?;
```

Comment:
I usually use `...)?.1` in such cases — doesn’t really offer less information to the reader

Reply:
:+1: